zerver/lib/test_classes.py

```diff
@@ -781,6 +781,34 @@ def assert_json_error(self, result: HttpResponse, msg: str, status_code: int=400
         """
         self.assertEqual(self.get_json_error(result, status_code=status_code), msg)
 
+    def get_all_json_errors(self, result: HttpResponse) -> List[str]:
+        """
+        Some API endpoints return multiple error messages and we need to
+        test for all the errors. Example JSON:
+        { "result":"error",
+          "msg":"This field cannot be blank.",
+          "errors":{
+            "pattern":["This field cannot be blank."],
+            "__all__":["No groups found in URL format string."]
+          }
+        }
+        """
+
+        try:
+            json = orjson.loads(result.content)
```

Comment:
`json` is the wrong variable name for something we’ve just translated *out* of JSON.